demos/improved_qadi_demo.py

```
@@ -0,0 +1,178 @@
+        
+        # Analyze question for specific topics
+        question_lower = question.lower()
+        
+        if "food waste at home" in question_lower:
+            return [
+                "Plan meals and shop with a list to buy only what you need",
+                "Store food properly using airtight containers and optimal temperatures", 
+                "Use leftovers creatively in new meals or freeze for later use"
+            ][:max_answers]
+        
+        elif "productivity" in question_lower and "working from home" in question_lower:
```

Ruff (0.11.9)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)